### PR TITLE
Feat: Implement workflow mutability policy

### PR DESCRIPTION
**Pull request by Zitzak**
## Implements
- https://linear.app/inverter/issue/SC-46/implement-workflow-mutability-policy

### CODE REVIEW

**Comment by marvinkruse** [comment]
Just did a first pass over the implementation. Looks good to me! Will do a second pass within the next 30 minutes, but haven't found anything yet - just fyi.

**Review by marvinkruse — APPROVED** [review]
lgtm - good work! 🙏

File: src/orchestrator/Orchestrator_v1.sol
Comment (FHieser) [inline src/orchestrator/Orchestrator_v1.sol:1]: Is it possible to remove the Authorizer via the ModuleManager directly and therefor brick it?
We might wanna change that in this PR too

**Review by FHieser — CHANGES_REQUESTED** [review]

File: src/orchestrator/Orchestrator_v1.sol
Comment (Zitzak) [reply src/orchestrator/Orchestrator_v1.sol:1]: Just for my understanding, you mean that we can remove the authorizer through the ModuleManager -> (initiate)removeModule and then we wouldn't be able to call the functions within the Orchestrator anymore that require authentification. Is that correct? @FHieser

Comment (marvinkruse) [reply src/orchestrator/Orchestrator_v1.sol:1]: Good point @FHieser - thanks for adding this @Zitzak! 🙏

**Review by marvinkruse — APPROVED** [review]
Good stuff, latest commit looks fine to me!

**Review by FHieser — APPROVED** [review]

**Review by marvinkruse — APPROVED** [review]

**Pull request merged**